authz/opa-bridge/README.md

```diff
@@ -0,0 +1,7 @@
+# Lakekeeper Open Policy Agent (OPA) Bridge
+The Lakekeeper OPA bridge is a gateway that allows trusted query engines to use Lakekeeper permissions via OPA. This allows query engine such as trino, which is shared by multiple users, to enforce permissions for individual users on objects governed by Lakekeeper. Currently trino is the only supported query engine, but we hope to see broader adoption in the future.
+
```

Comment:
could we add a small section about each of the directories in here? E.g. what's in inputs & policies & policies subdirs?

Reply:
Done